superbench/analyzer/file_handler.py

```diff
@@ -0,0 +1,154 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+"""A module for data analysis."""
+
+from pathlib import Path
+import re
+
+import json
+import jsonlines
+import pandas as pd
+import yaml
+
+from superbench.common.utils import logger
+
+
+def read_raw_data(raw_data_path):
+    """Read raw data from raw_data_path and store them in self._raw_data_df.
+
+    Args:
+        raw_data_path (str): the path of raw data jsonl file
+    Returns:
```

Comment:
Blank line between `Args` and `Returns`.

Reply:
ok